.env

```diff
@@ -1,4 +1,5 @@
 NEOGO_VERSION=0.99.3
-AIO_VERSION=0.32.0
+AIO_VERSION=0.34.0
```

Comment:
Should we wait for https://github.com/nspcc-dev/neofs-aio/pull/39?

Reply:
Forgot to mark as draft.